tests/openvino/test_modeling.py

```diff
@@ -1037,6 +1037,7 @@ class OVModelForCausalLMIntegrationTest(unittest.TestCase):
         "decilm",
         "minicpm3",
         "deepseek",
+        "phi3-moe",
```

Comment:
why ? it's supported in transformers since 4.46 no ?

Reply:
updated